### PR TITLE
added llvm 7.0 support

### PR DESCRIPTION
**Pull request by foerdi**
added llvm 7.0.0 support based on the git diffs between release_60 and release_70.

### CODE REVIEW

File: source/llvm/constants.d [hunk 1]
```diff
@@ -272,6 +272,53 @@ static if (LLVM_Version >= asVersion(3, 5, 0))
 }
 
 static if (LLVM_Version >= asVersion(3, 4, 0))
```
Comment (MoritzMaxeiner) [inline source/llvm/constants.d]: I think this should be changed to `static if (LLVM_Version >= asVersion(7, 0, 0))`.

File: source/llvm/constants.d [hunk 2]
```diff
@@ -654,6 +743,10 @@ static if (LLVM_Version >= asVersion(6, 0, 0))
 		                           LLVMDIFlagMultipleInheritance |
 		                           LLVMDIFlagVirtualInheritance
 	}
+} 
+
+static if (LLVM_Version >= asVersion(6, 0, 0))
```
Comment (MoritzMaxeiner) [inline source/llvm/constants.d]: Why this change, are you saying `LLVMDWARFSourceLanguage`, which is already in llvm-d was wrongly supported for LLVM < 6?

File: source/llvm/constants.d (hunk 1 above)
Comment (foerdi) [reply source/llvm/constants.d]: ty, is fixed

File: source/llvm/constants.d (hunk 2 above)
Comment (foerdi) [reply source/llvm/constants.d]: `LLVMDWARFSourceLanguage` was not changed, before it was in a big `static if (LLVM_Version> = asVersion (6, 0, 0))` block. Now I have split it.

Comment (MoritzMaxeiner) [reply source/llvm/constants.d]: Ah, didn't catch that, thanks
